src/utils/documentdb_model.ts

```diff
@@ -47,25 +49,22 @@ export abstract class DocumentDbModel<
    *     resolves to either an error or the retrieved version of the persisted
    *     document.
    */
-  public async create(
-    document: TN,
-    partitionKey: string
-  ): Promise<Either<DocumentDb.QueryError, TR>> {
+  public create(document: TN): TaskEither<ErrorResponse, TR> {
     // we don't want to store the "kind" property, let's remove it
+    // FIXME: is this filter still needed?
```

Comment:
@gunzip @francescopersico do we still need to remove the `kind` attribute before storing objects on cosmos?